src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -7,8 +7,32 @@
 public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
```

Comment:
It's better to inject StorageDao as a dependency through the constructor. This will make your code more flexible and easier to test. It's a good practice to follow the Dependency Inversion Principle, which is one of the SOLID principles.